keps/sig-apps/2214-indexed-job/README.md

```diff
@@ -424,64 +528,60 @@ _This section must be completed when targeting beta graduation to a release._
 
 * **What specific metrics should inform a rollback?**
 
+  - job_sync_duration_seconds shows significantly more latency for Indexed Jobs.
```

Comment:
Do we have a label representing job type? If so, can you make it explicit (also provide label name).

Reply:
this was already updated in the parent PR #2616 :)

Rebased.